docs/modules/plugins/pages/plugin-excel.adoc

```diff
@@ -74,3 +74,35 @@ Examples:
 Examples:
 {transformer=FROM_EXCEL, path=/my-excel-file.xlsx, sheet=Data, range=A2:B4}
 ----
+== Steps
+
+=== Create excel file with specific content
+
+==== *_Info_*
```

Comment:
please update the format of the docs and use https://docs.vividus.dev/vividus/latest/plugins/plugin-aws-s3.html as example